packages/rainbowkit/src/hooks/useMainnetEnsAvatar.ts

```diff
@@ -6,9 +6,17 @@ import { useIsMainnetConfigured } from './useIsMainnetConfigured';
 export function useMainnetEnsAvatar(name: GetEnsNameReturnType | undefined) {
   const mainnetConfigured = useIsMainnetConfigured();
 
+  const safeNormalize = (ensName: string) => {
+    try {
+      return normalize(ensName);
+    } catch {
+      return ensName;
+    }
+  };
+
   const { data: ensAvatar } = useEnsAvatar({
     chainId: mainnet.id,
-    name: name ? normalize(name) : undefined,
+    name: name ? safeNormalize(name) : undefined,
```

Comment:
I think this would still fail because the raw `ensName` returned wouldn't be normalized and then `useEnsAvatar` would fail in a cascade. I think we should either just get the normalization fixed upstream in viem or skip ENS resolution when `normalize` throws an error.
<img width="721" alt="Screenshot 2024-03-19 at 2 02 55 AM" src="https://github.com/rainbow-me/rainbowkit/assets/4412473/c3c7b739-5a8e-4767-aba9-585b0571797c">